666 get the components to install the components + add glossary of terms to docs (#884)
mi6/ic-design-system: src/content/structured/community/contribute.mdx

1. Select a ticket from the [Design System](https://github.com/mi6/ic-design-system/issues) or the [UI Kit](https://github.com/mi6/ic-ui-kit/issues) backlog.
2. If a backlog ticket doesn’t exist for your contribution, you can suggest it through GitHub [Design System](https://github.com/mi6/ic-design-system/issues) or [UI Kit](https://github.com/mi6/ic-ui-kit/issues) issues, or our internal Service Desk.
3. If you're an internal user, we will arrange a kickoff meeting to discuss the scope, plan and agree any support. For any external users, we'll work with you through GitHub issues.

> 4. Refer to the technical instructions for [Design System](https://github.com/mi6/ic-design-system/tree/main/CONTRIBUTING.md) or the [UI Kit repository](https://github.com/mi6/ic-ui-kit/blob/main/CONTRIBUTING.md) coding standards and practices. If you're contributing a feature, you might want to read through the [glossary of terms](/community/glossary-of-terms) to help with prop/slot naming.

Reviewer: ```suggestion
4. Refer to the technical instructions for [Design System](https://github.com/mi6/ic-design-system/tree/main/CONTRIBUTING.md) or the [UI Kit repository](https://github.com/mi6/ic-ui-kit/blob/main/CONTRIBUTING.md) coding standards and practices. If you're contributing a feature, we recommend reading the [glossary of terms](/community/glossary-of-terms) to help with prop/slot naming.
```
